index.js

```diff
@@ -32,8 +32,19 @@ app.get('/todos', (request, response) => {
   });
 });
 
+// Update the title via server:port/todos/id/new_title
+app.patch('/todos/:id/:title', (request, response) => {
```

Comment:
hey Jack your function will work, but I don't see the need of the `:title` parameter so you need only the id which supplied by the user, and also to be consistent you should replace `todos` to **todo** instead,
because in fact you're changing(updating) only one todo..